app/assets/javascripts/services/deploy_helpers_factory.js

```diff
@@ -0,0 +1,26 @@
+samson.factory('DeployHelper', function($window, $log, Deploys) {
```

Comment:
oh you just moved that around ... still not a fan but ... never mind if it's old ...